### DISCUSSION TITLE
Bump h2 from 1.4.200 to 2.1.210 (#251, #256)

### DISCUSSION THREAD

**Discussion by DushyantTa** [OpenAS2/OpenAs2App#264] [Q&A] [answered]
How to Migrate H2 1.4.200 to 2.1.210?
Please update openas2.mv.db as well as db_ddl.sql files.

**Comment by ghost** [comment, marked as answer]
[Fix for h2 migration from 1.4.200 to 2.1.210 (](https://github.com/OpenAS2/OpenAs2App/commit/af8180c64f6f12b18e048c2bb8dd25b62a58827f)https://github.com/OpenAS2/OpenAs2App/pull/265[)](https://github.com/OpenAS2/OpenAs2App/commit/af8180c64f6f12b18e048c2bb8dd25b62a58827f)
